ext/phar/stream.c

```diff
@@ -229,7 +229,7 @@ static php_stream * phar_wrapper_open_url(php_stream_wrapper *wrapper, const cha
 			}
 		}
 		if (opened_path) {
-			*opened_path = strpprintf(MAXPATHLEN, "phar://%s/%s", idata->phar->fname, idata->internal_file->filename);
+			*opened_path = zend_strpprintf_unchecked(MAXPATHLEN, "phar://%s/%S", idata->phar->fname, idata->internal_file->filename);
```

Comment:
It feels like a string with a null byte should be caught earlier, but I know this is very inconsistently checked in ext/phar, nor does this change really hurt.

Reply:
This change isn't really about null bytes, it's just for convenience so I don't have to add ZSTR_VAL for the filename field

Reply:
ack